### PR TITLE
treewide: update thingy91x bootloader setup

### PR DESCRIPTION
**Pull request by maxd-nordic**
Use production setup of thingy91x bootloader configuration.

### CODE REVIEW

File: samples/cellular/nrf_cloud_multi_service/sysbuild/mcuboot/boards/thingy91x_nrf9151.conf [hunk 1]
```diff
@@ -14,3 +14,8 @@ CONFIG_UART_CONSOLE=n
 CONFIG_MCUBOOT_SERIAL=y
 CONFIG_MCUBOOT_SERIAL_DIRECT_IMAGE_UPLOAD=y
 CONFIG_BOOT_SERIAL_IMG_GRP_IMAGE_STATE=y
+
```
Comment (plskeggs) [inline samples/cellular/nrf_cloud_multi_service/sysbuild/mcuboot/boards/thingy91x_nrf9151.conf:17]: Seems ok. @tony-le-24 can you check if the various additional nightly and weekly tests still work with this change -- any of them that run on the Thingy:91X?

Comment (tony-le-24) [reply samples/cellular/nrf_cloud_multi_service/sysbuild/mcuboot/boards/thingy91x_nrf9151.conf:17]: I will, thanks

Comment (tony-le-24) [reply samples/cellular/nrf_cloud_multi_service/sysbuild/mcuboot/boards/thingy91x_nrf9151.conf:17]: I ran it on my PR for Thingy91:X testing, things seem green. LGTM!

**Review by tony-le-24 — APPROVED** [review]

**Comment by nordicjm** [comment]
sdk-mcuboot PR has been merged, have brought in manifest update and commit here into https://github.com/nrfconnect/sdk-nrf/pull/17567 to save on CI

**Pull request closed without merging**